### ISSUE TITLE
Java Token Failure

### ISSUE THREAD

**Issue by ishc** [amzn/alexa-avs-raspberry-pi#212]
I got everything working but have ran into some frustrating issues in the home stretch.
I have my authentication token in the Alexa Voice Service, it successfully records my voice input, but doesn't answer. Instead, I get an error saying:
`java.util.concurrent.ExecutionException: Java.nio.channels.ClosedChannelException`
(Screenshot: http://prntscr.com/bhogan)
I tried doing some research as to what this error might mean but I can't seem to crack it.
Have any of you ran into this problem as well?
Any ideas on how this might be fixed?

Cheers.

**Comment by ajotwani** [comment]
@ishc just checking, you still having issues? We've updated this project to include hands-free voice activation, using wake word engines from Sensory or KITT.AI.

So, in addition to pushing a button to "start listening", you can now also just say the wake word "Alexa", much like the Amazon Echo. Perhaps give it another whirl, and let us know how it goes?

**Comment by notvishwanth** [comment]
@ajot hey, with regards to the issue, i'm getting this issue right now, could you help us out?

**Comment by keleric** [comment]
You'll often get an AsynchronousCloseException if you try to use 2 clients at the same time with the same ProductID & DSN. Only 1 client at a time can use that pairing. Can you ensure verify that you're only using 1 device?

**Comment by notvishwanth** [comment]
@keleric , yes i am running the project on cloud too, but I run one project at a time

**Comment by keleric** [comment]
Also, can you verify that the time zone on your raspberry pi matches what AVS expects it to be in alexa.amazon.com?

**Comment by shreyasnaik** [comment]
I'm getting the same issue and I've checked time zones match; any  other suggestions?
